src/manage_nsfs/manage_nsfs_cli_errors.js

```diff
@@ -490,6 +490,16 @@ ManageCLIError.ConfigDirUpdateBlocked = Object.freeze({
     http_code: 500,
 });
 
+///////////////////////////////
+//     CONNECTION ERRORS     //
+///////////////////////////////
+
+ManageCLIError.MissingCliParam = Object.freeze({
+    code: 'MissingCliParam',
+    message: 'Connection name is mandatory, please use the --name flag',
```

Comment:
I saw that you reused this error also for other properties, so you need to update the message here, maybe use helper function.